src/client.cpp

```diff
@@ -688,4 +688,5 @@ void PauseStream(bool _UNUSED(bPaused)) {}
 bool SeekTime(int, bool, double*) { return false; }
 void SetSpeed(int) {};
 bool IsTimeshifting(void) { return false; }
+bool IsRealTimeStream(void) { return true; }
```

Comment:
> As explained in the dvbviewer subforum (you probably haven't read this, but @Uukrull might have), I'll adapt the PVR in the next days.

I didn't read that. If you're going to update the PVR with the API changes, this PR can be closed.